addon/page-object.js

```diff
@@ -87,6 +84,10 @@ export default class PageObject {
     });
   }
 
+  clickWithText(rawSelector, text) {
```

Comment:
Not crazy about `with` as it seems to imply that the verb is carrying something with it. I think `by`is more expressive (and shorter):

`clickByText`, `findByText`, `fillInByLabel`, etc.